tests/test_bounds.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# This script is a draft. Note: Bash is not available on CheriBSD.
+# This is a simple way to test the examples by checking ``stderr``.
+
+: "${SSHPORT:=10021}"
+RESULT_riscv64="fail"
+if [ -f "../bin/bounds" ]; then
+    scp -P $SSHPORT ../bin/bounds root@127.0.0.1:/root
+	RESULT_riscv64=`ssh -p $SSHPORT root@127.0.0.1 -t 'sh -c "/root/bounds 2>&1 > /dev/null"'`
+    echo $RESULT_riscv64
+    # note: other forms of equality (test) fail on SH
+    if [ -z "${RESULT_riscv64##*In-address space security exception (core dumped)*}" ] ;then
+        echo "TEST PASSED!"
+    else
+        echo "TEST FAILED!"
+        echo "Output: $RESULT_riscv64"
+fi
```

Comment:
Wrong indentation.

Reply:
That's weird. I can't see even the previous wrong indentation in my editor :|

Reply:
There's a mix of tabs and spaces -- I strongly suggest only using spaces to avoid this problem.

Reply:
That's a common problem with VSCode but I thought I had solved it.